networking/eth2/src/main/java/tech/pegasys/teku/networking/eth2/rpc/beaconchain/methods/DataColumnSidecarsByRootMessageHandler.java

```diff
@@ -102,23 +105,21 @@ public void onIncomingMessage(
       final String protocolId,
       final Eth2Peer peer,
       final DataColumnSidecarsByRootRequestMessage message,
-      final ResponseCallback<DataColumnSidecar> callback) {
+      final ResponseCallback<DataColumnSidecar> responseCallback) {
 
-    long requestId = REQUEST_ID_COUNTER.getAndIncrement();
+    ReqRespResponseLogger<DataColumnSidecar> responseLogger =
+        dasLogger
+            .getDataColumnSidecarsByRootLogger()
+            .onInboundRequest(
+                LoggingPeerId.fromPeerAndNodeId(
+                    peer.getId().toBase58(), peer.getDiscoveryNodeId().orElseThrow()),
+                message.asList());
 
-    LOG.debug(
-        "Peer {} requested {} data column sidecars with identifiers: {}",
-        peer.getId(),
-        message.size(),
-        message);
-    LOG_DAS.info(
-        "[nyota] DataColumnSidecarsByRootMessageHandler: REQUEST(#{}) {} data column sidecars from {}",
-        requestId,
-        message.size(),
-        peer.getId());
+    LoggingResponseCallback<DataColumnSidecar> loggingCallback =
+        new LoggingResponseCallback<>(responseCallback, responseLogger);
 
     final Optional<RequestApproval> dataColumnSidecarsRequestApproval =
-        peer.approveDataColumnSidecarsRequest(callback, message.size());
+        peer.approveDataColumnSidecarsRequest(loggingCallback, message.size());
```

Comment:
same here with `loggingCallback`, I'd prefer to hide logging or put it in 2nd role in name